inst/examples/router/app.r

```diff
@@ -1,90 +1,190 @@
-    f7Route(other_page, "/other", main = FALSE, name = "other")
-)
-router[[2]][[2]]$attribs$class <- paste(
-    router[[2]][[2]]$attribs$class,
-    "views tabs"
-)
-
-ui <- f7Page(
-    title = "My app",
-    options = list(
-        dark = TRUE,
-        routes = list(
-            list(path = "/", url = "/"),
-            list(path = "/other", url = "/other")
+page_2 <- function() {
+  page(
+    href = "/2",
+    ui = function(request) {
+      shiny::tags$div(
+        class = "page",
+        # top navbar goes here
+        f7Navbar(
+          "Second page",
+          # Allows to go back to main
+          leftPanel = tags$a(
+            href = "/",
+            class = "link",
+            tags$i(class = "icon icon-back"),
+            tags$span(
+              class = "if-not-md",
+              "Back"
+            )
+          )
+        ),
+        f7Toolbar(
+          position = "bottom",
+          tags$a(
+            href = "/",
+            "Main page",
+            class = "link"
+          )
+        ),
+        shiny::tags$div(
+          class = "page-content", f7Block(
+            strong = TRUE,
+            inset = TRUE,
+            outline = TRUE,
+            f7Block(f7Button(inputId = "update", label = "Update stepper")),
+            f7Block(
+              strong = TRUE,
+              inset = TRUE,
+              outline = TRUE,
+              f7Stepper(
+                inputId = "stepper",
+                label = "My stepper",
+                min = 0,
+                max = 10,
+                size = "small",
+                value = 4,
+                wraps = TRUE,
+                autorepeat = TRUE,
+                rounded = FALSE,
+                raised = FALSE,
+                manual = FALSE,
+                layout = "list"
+              ),
+              verbatimTextOutput("test")
+            )
+          )
         )
-    ),
-    router
-)
+      )
+    }
+  )
+}
 
-server <- function(input, output, session) {
-    router_server()
+page_3 <- function() {
+  page(
+    href = "/3",
+    ui = function(request) {
+      shiny::tags$div(
+        class = "page",
+        # top navbar goes here
+        f7Navbar(
+          "Third page",
+          # Allows to go back to main
+          leftPanel = tags$a(
+            href = "/",
+            class = "link",
+            tags$i(class = "icon icon-back"),
+            tags$span(
+              class = "if-not-md",
+              "Back"
+            )
+          )
+        ),
+        f7Toolbar(
+          position = "bottom",
+          tags$a(
+            href = "/2",
+            "Second page",
+            class = "link"
+          )
+        ),
+        shiny::tags$div(
+          class = "page-content",
+          f7Block("Nothing to show yet ...")
+        )
+      )
+    }
+  )
 }
 
-shinyApp(ui, server)
+brochureApp(
+  # Pages
+  page_1(),
+  page_2(),
+  page_3(),
+  wrapped = f7Page,
+  wrapped_options = list(
```

Comment:
@hypebright and @ColinFay I had to hack brochure to pass a wrapper with extra [options](https://github.com/DivadNojnarg/brochure/commit/a96e006471d15acab87b76b4e9ae8ea1c08842ae#diff-9f6ec8a9e4d6e9199096fc4d6e7c524eec810f0273b84e39460e1206d03a2a3eR44)